.buildkite-bazelrc

```diff
@@ -12,7 +12,8 @@
 #build:remote-cache --disk_cache=
 build:remote-cache --remote_download_toplevel
 build:remote-cache --remote_cache=grpc://bazel-remote-cache:9092
-build:remote-cache --experimental_remote_downloader=grpc://bazel-remote-cache:9092
+# Does not work with rules_oci.
+#build:remote-cache --experimental_remote_downloader=grpc://bazel-remote-cache:9092
```

Comment:
this is now commented out right? what does this do?

Reply:
Yeah, it doesn't work well with rules_oci. I'll amend the comment above to link to the issue. Basically, the method of pulling base images is non-hermetic and is causing issues in CI (i think). No confident in the root cause this flag is problematic with rules_oci.

https://github.com/bazel-contrib/rules_oci/issues/292

Reply:
Add ref to the above issue in the commentary.